float32 time
float32 left_pwm
float32 right_pwm
float32 left_pressure
float32 right_pressure
float32 left_flex
float32 right_flex
geometry_msgs/Vector3 base_pos
geometry_msgs/Vector3 tip_pos